pkg/virt-api/rest/subresource_test.go

```diff
@@ -342,18 +330,6 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 			close(done)
 		}, 5)
 
-		It("should fail to connect to the serial console if the VMI is paused", func(done Done) {
```

Comment:
same as above

Reply:
Those unit tests was testing specific code in subresource.go handlers. I removed that code so I think it's ok to remove the unit tests. Handler functions are still covered by other tests in subresource_test.go

Reply:
I see your point.
I personally think that it's better to keep the tests to ensure that it's possible to connect to paused VMs. After all you didn't delete code because it is not relevant but to allow a new capability which IMO needs to be checked in tests.

On the other hand it is being checked via functional tests so I wouldn't say it's a must. Your call :)